docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -806,4 +806,163 @@ The following is performed on ArduCopter version v4.5.7
+
+*************************************
+Calculating DroneCAN Utilization
+*************************************
+
+Background
+============
+The basic premise of calculating DroneCAN bus utilization is to figure out how much time you spend sending DroneCAN messages compared to some period of time. For example, 
+suppose your message takes 500ms to transmit. Then, in a 1 second period of time, that message takes up 50% of the available bandwidth.
+
+The basic calculation we are going to perform is summarized in the image below:
+
+.. image:: ../_static/comms_protocols_pictures/dronecan/dronecan_utilization_calculation.png
+
+DroneCAN frames are broken down as follows:
+
+.. image:: ../_static/comms_protocols_pictures/dronecan/dronecan_packet.png
+
+Please note that the Data bytes may be split further depending on whether or not the current frame is the beginning of a multi frame transfer. As specified by the 
+`DroneCAN specification <https://dronecan.github.io/Specification/4.1_CAN_bus_transport_layer/#:~:text=is%20not%20suppressed.-,Payload,-The%20Data%20field>`_, the data may be broken down as either: 
+
+.. image:: ../_static/comms_protocols_pictures/dronecan/single_frame_transfer.png
+
+or
+
+.. image:: ../_static/comms_protocols_pictures/dronecan/multi_frame_transfer.png
+
+With no data (and thus no tail byte), this is 67 bits and 134us, and with a full 7 bytes of data (plus tail byte), this is 131 bits or 256us.
+
+You can read more about the DroneCAN transport layer in the `DroneCAN spec <https://dronecan.github.io/Specification/4.1_CAN_bus_transport_layer/>`_.
+
+We can now easily apply this information to different types of DroneCAN messages. Shown here are the impacts of sending DroneCAN telemetry and RawCommands.
+
+.. _telemetry_utilization:
+
+Utilization With DroneCAN Telemetry
+=======================================
+
+By default, Vertiq modules transmit both the DroneCAN StatusExtended message as well as the EscStatus message.
+
+Each `StatusExtended <https://github.com/dronecan/DSDL/blob/master/uavcan/equipment/esc/1036.StatusExtended.uavcan>`_ message sends exactly 56 bits of data, and only requires one full frame. This gives us about 131 bits of data.
+
+Each `EscStatus <https://github.com/dronecan/DSDL/blob/master/uavcan/equipment/esc/1034.Status.uavcan>`_ message transmits 110 bits of data, and requires 3 frames to transmit. 
+The first two frames send a full 7 bytes of data (plus the tail byte, and including the 2 CRC bytes in the first frame as discussed above), and the final frame sends only 3 
+bytes (including the tail byte). This means the first two frames are about 131 bits, and the final frame will have 91 bits.
+
+An example of a Vertiq module transmitting telemetry as viewed through the DroneCAN GUI tool is given below.
+
+.. image:: ../_static/comms_protocols_pictures/dronecan/telem_image.png
+
+In total, Vertiq module's DroneCAN telemetry requires:
+
+131 bits StatusExtended + 353 bits EscStatus = 484 bits.
```

Comment:
Stylistic thing, I feel like this may benefit from being italicized or a different kind of font or something to separate it out as a bit of math from the surrounding text. Kind of all just flows together right now, good to have the math be a bit distinct I think. Same goes for all equation stuff in the middle of blocks of text